.github/workflows/colima-tests.yml

```diff
@@ -108,7 +97,7 @@ jobs:
 
       - name: Homebrew cache/restore
         uses: actions/cache/restore@v3
-        if: ${{ github.ref != 'refs/heads/master' }}
+        if: github.ref != 'refs/heads/master'
```

Comment:
I'm always baffled by the mixed use of golang templates and direct comparisons in Github actions.

Reply:
Don't get it, these are JS conditions. `${{ }}` is the so called expression syntax which will be replaced by the runner before running the workflow. But as `if` is always assumed to be a condition the expression syntax is superfluous but works for compatibility.

Reply:
It's actually a `$` with golang template.

Reply:
At least that's what I think!